### PR TITLE
Example of using xref

### PR DESCRIPTION
**Pull request by marcoscaceres**
(don't merge this :) ... just examples)

### CODE REVIEW

File: common/terms.html [hunk 1]
```diff
@@ -1,343 +1,348 @@
...
+      A single colon comes after each name,
+      separating the name from the value.
+      A single comma separates a value from a following name.
+      In JSON-LD the names in an object MUST be unique.
+      <p>In the <a>internal representation</a> a <a>JSON object</a> is described as a
+        <dfn data-cite="INFRA#ordered-map" class="preserve">map</dfn> (see [[INFRA]]),
+        composed of <dfn data-cite="INFRA#map-entry" data-lt="map entry|entry" data-lt-noDefault class="preserve">entries</dfn> with key/value pairs.</p>
+      <p>In the <a data-cite="JSON-LD11-API#the-application-programming-interface">Application Programming Interface</a>,
+        a <a>map</a> is described using a [[WEBIDL]] <a>dictionary</a>.</p></dd>
+    <dt class="changed">JSON-LD <a>internal representation</a></dt>
```
Comment (marcoscaceres) [inline common/terms.html]: Here is an example of xref auto-linking a common term.

(I think this document has Window's line endings, which Visual Studio Code removed)

File: common/terms.html [hunk 2]
```diff
@@ -1,343 +1,348 @@
...
+      A name is a <a>string</a>.
+      A single colon comes after each name,
+      separating the name from the value.
+      A single comma separates a value from a following name.
+      In JSON-LD the names in an object MUST be unique.
+      <p>In the <a>internal representation</a> a <a>JSON object</a> is described as a
+        <dfn data-cite="INFRA#ordered-map" class="preserve">map</dfn> (see [[INFRA]]),
+        composed of <dfn data-cite="INFRA#map-entry" data-lt="map entry|entry" data-lt-noDefault class="preserve">entries</dfn> with key/value pairs.</p>
+      <p>In the <a data-cite="JSON-LD11-API#the-application-programming-interface">Application Programming Interface</a>,
+        a <a>map</a> is described using a [[WEBIDL]] <a>dictionary</a>.</p></dd>
```
Comment (marcoscaceres) [inline common/terms.html]: Note here, just `<a>dictionary</a>`... don't need data-cite.

File: common/terms.html [hunk 3]
```diff
@@ -1,343 +1,348 @@
...
+    <dt class="changed">JSON-LD <a>internal representation</a></dt>
+    <dd class="changed">
+      The JSON-LD internal representation
+      is the result of transforming a JSON syntactic structure
+      into the core data structures suitable for direct processing:
+      <a>arrays</a>, <a>maps</a>, <a>strings</a>, <a>numbers</a>, <a>booleans</a>, and <a>null</a>.</dd>
+    <dt>
+      <a>null</a>
```
Comment (marcoscaceres) [inline common/terms.html]: Similarly here... just `<a>null</a>`.

File: common/terms.html [hunk 4]
```diff
@@ -1,343 +1,348 @@
...
-    which is a <a>string</a>,
-    and a type,
-    which is an <a>IRI</a>.</dd>
-  <dt><dfn>value object</dfn></dt><dd>
-    A <a>value object</a> is a <a>map</a> that has an <code>@value</code> <a>entry</a>.</dd>
-  <dt><dfn class="preserve">vocabulary mapping</dfn></dt><dd>
-    The vocabulary mapping is set in the <a>context</a> using the <code>@vocab</code> key
-    whose value MUST be an <a>IRI</a> or <code>null</code>.</dd>
-</dl>
+    <dt><dfn data-cite="INFRA#list" class="preserve">array</dfn></dt><dd>
```
Comment (marcoscaceres) [inline common/terms.html]: Note that this might confuse people... array and lists are different things.

Comment (gkellogg) [reply common/terms.html]: Yes, but it was not feasible to change existing uses of list, which are associated with RDF lists and oradered arrays in JSON-LD. Array is also more compatible with general JSON and WebIDL nomenclature.

File: common/terms.html (hunk 3 above)
Comment (gkellogg) [reply common/terms.html]: It doesn't work for me, I get "Couldn't match "null" to anything in the document or to any other spec."

**Pull request closed without merging**